tests/integration/012_environment/main.tf

```diff
@@ -34,11 +33,6 @@ resource "env0_environment" "example" {
   vcs_commands_alias         = "alias"
 }
 
-data "env0_configuration_variable" "env_config_variable" {
```

Comment:
This was added in the original commit. I guess it no longer works properly without waiting.
Had to remove it for the test to pass.